apps/molgenis-components/src/components/forms/BaseInput.vue

```diff
@@ -2,24 +2,44 @@
 export default {
   name: 'BaseInput',
   props: {
+    /**
+     * Unique identifier https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id
+     * Has no direct emx2 implementenation but can be constructend by concatinating the table name with the field name
+     */
     id: {
       type: String,
       required: true
     },
+    /**
+     * Name of the element. For example used by the server to identify the fields in form submits.
+     * relates to emx 2 columnName
+     */
     name: {
       type: String,
       required: false,
       default: function () { return this.id }
     },
     value: {type: [String, Number, Object, Array, Boolean], default: null},
+    /**
+     * Caption for an item in a user interface.
+     * relates to emx 2 columnName
+     */
     label: {
       type: String,
       required: false
     },
+    /**
+     * Additional information describing the field 
+     * relates to emx 2 description
+     */
```

Comment:
why don't we then call it 'description'?

Reply:
We use bootstap as  a htm css basis for our components , bootstrap calls this 'help' , imo its also more semantic as the label should describe the field and the help text can optionally add some help where needed ( like a comment). we could call it description. probably the route cause lies with emx mixing the 'desciption' and 'identified' into a single name  field and then needing a description to describe the field

Reply:
please call it 'description'. Only alternative is that we discuss with DM if we gonna rename 'description' to 'help' but I don't think that is an improvement

NB our focus is not 'bootstrap' but 'emx2' so we try to adapt bootstrap to emx2 instead of adapting emx2 to bootstrap. 
NB2 'description' is not the same a 'name' so it seems we have different understanding of emx2 metadata definition.  If you want to say something about emx 'mixing' then it is that emx2 tries to stop confusion between different 'name', 'identifier' and 'label' for same field, by saying you can have only one 'name' for a field. 'description' has the purpose to attach a text length explanation to that field. This could be in the form of a question, or in the form of an explanation.